themes/classic/templates/errors/not-found.tpl

```diff
@@ -24,9 +24,17 @@
  *}
 <section id="content" class="page-content page-not-found">
   {block name='page_content'}
+    {if isset($title)}
+      <h4>{block name="searchTitle"}{$title}{/block}</h4>
+    {else}
+      <h4>{block name="searchTitle"}{l s='Sorry for the inconvenience.' d='Shop.Theme.Global'}{/block}</h4>
```

Comment:
Hi! These strings ("Sorry for the inconvenience" and "Search again what you are looking for." were supposed to be canceled and replaced by the new wordings.

Reply:
Oh yes, my bad